configuration/defaults.go

```diff
@@ -69,4 +69,6 @@ func SetDefaults(settings *viper.Viper) {
 	settings.BindEnv("directories.Downloads", "ARDUINO_DOWNLOADS_DIR")
 	settings.BindEnv("directories.Data", "ARDUINO_DATA_DIR")
 	settings.BindEnv("sketch.always_export_binaries", "ARDUINO_SKETCH_ALWAYS_EXPORT_BINARIES")
+
+	settings.SetDefault("sketch.template", "")
```

Comment:
This didn't fit with the other settings, not sure if I should create a section here like `// misc sketch settings`.

Reply:
Nope just keep it there, we will adjust the ordering in the future if more settings will come up.